Library/Homebrew/cmd/update.sh

```diff
@@ -22,22 +22,42 @@ git() {
 git_init_if_necessary() {
   if [[ -n "$HOMEBREW_OSX" ]]
   then
-    OFFICIAL_REMOTE="https://github.com/Homebrew/brew.git"
+    BREW_OFFICIAL_REMOTE="https://github.com/Homebrew/brew"
+    CORE_OFFICIAL_REMOTE="https://github.com/Homebrew/homebrew-core"
   else
```

Comment:
Don't need to fix this now but I'd actually check for Linux and otherwise do just `Homebrew/homebrew-core` so we don't assume non-OS X == Linux.